### PR TITLE
Cleans up telescience computer code. All of it.

### PR DESCRIPTION
**Pull request by PJB3005**
Also removes the non-nanoUI option of the telescience computer. It's just too high maintenance to bother supporting.

This god damn code man.

This PR was prompted by me getting severely annoyed by a refresh issue with the telescience console. It's fixed now.

### CODE REVIEW

**Comment by ghost** [comment]
Literally the only console that allows you to choose nanoUI or default UI has default UI removed.

The midas touch of shit: Everything PJB touches turns into nano.

**Comment by PJB3005** [comment]
@Unknown-as-Captain I coded that system originally. NanoUI is 100% the only way to go. The other system was just shitty and not worth maintaining.

File: code/modules/telesci/telesci_computer.dm [hunk 1]
```diff
@@ -41,74 +43,72 @@
...
-			if(areaMaster.power_equip == 0) // There's no APC in this area, don't try to cheat power!
-				to_chat(user, "<span class='warning'>\The [name] blinks red as you try to insert the cell!</span>")
-				return
+	if(cell)
+		to_chat(user, "<span class='warning'>There is already a cell in \the [name].</span>")
+		return TRUE
+
+	if(user.drop_item(W, src))
+		cell = W
+		user.visible_message("[user] inserts a cell into \the [src].", "You insert a cell into \the [src].")
```
Comment (ShiftyRail) [inline code/modules/telesci/telesci_computer.dm]: Shouldn't it be `\The [user]` ?

Comment (ShiftyRail) [inline code/modules/telesci/telesci_computer.dm:89]: Shouldn't it be `\The [user]` ?

Comment (PJB3005) [reply code/modules/telesci/telesci_computer.dm:89]: No, `\The` is implied so it doesn't matter, we only do it for readability, and in most cases user is a proper noun.

**Pull request merged**